test/e2e/assignments.txt

```diff
@@ -0,0 +1,116 @@
+**************** Sig assignments for tests ********************
+monitoring.go ('sig-ui', 2)
+deployment.go ('sig-scheduling', 4)
```

Comment:
I'm open to any algorithm for segregation, including manual curation - seems like that would be a pretty massive PR though ?